avocado/core/remote/runner.py

```diff
@@ -86,7 +89,13 @@ def check_remote_avocado(self):
         if result.exit_status == 127:
             return (False, None)
 
+        # Get the remote and local versions of avocado, and compare the major versions only
```

Comment:
`check_remote_avocado()` already returns the remote avocado version. You can keep it as is and handle the comparison in `run_suite()`.